docs/production/deployment.md

```diff
@@ -223,28 +223,14 @@ behind reverse proxies.
 
 [using-http]: ../production/deployment.html#configuring-zulip-to-allow-http
 
-## Using an outgoing HTTP proxy
+## Customizing the outgoing HTTP proxy
```

Comment:
We probably want a sentence that says "In Zulip 4.7 and older ..." or something.

Reply:
Something like?

```diff
diff --git docs/production/deployment.md docs/production/deployment.md
index f7acb5cde9..54d0093936 100644
--- docs/production/deployment.md
+++ docs/production/deployment.md
@@ -260,6 +260,10 @@ list][smokescreen-acls]. By default, Smokescreen denies access to all
 addresses](https://en.wikipedia.org/wiki/Private_network), including
 127.0.0.1, but allows traffic to all public Internet hosts.

+In Zulip 4.7 and older, to enable SSRF protection via Smokescreen, you
+will need to explicitly add the `zulip::profile::smokescreen` Puppet
+class, and configure the `[http_proxy]` block as above.
+
 [smokescreen]: https://github.com/stripe/smokescreen
 [smokescreen-acls]: https://github.com/stripe/smokescreen#acls
 [ssrf]: https://owasp.org/www-community/attacks/Server_Side_Request_Forgery
 ```

Reply:
Yeah, that works!  Though we could also mention Smokescreen is new in 4.0 somehow.